environment.yml

```diff
@@ -1,8 +1,10 @@
 name: example-environment
 channels:
+  - https://repo.mamba.pm/emscripten-forge
```

Comment:
Tracking whether this could be handled by `jupyterlite-xeus` in https://github.com/jupyterlite/xeus/issues/76